templates/base.tpl

```diff
@@ -53,25 +53,29 @@
     <hr/>
 
     <footer class="row">
-        <p class="col-md-6">
-            <small>
+        <div class="col-md-6">
+            <p>
                 Account Creation Assistance Manager
                 (<a href="https://github.com/enwikipedia-acc/waca/tree/{$toolversion}">version {$toolversion}</a>)
                 by <a href="{$baseurl}/internal.php/team">The ACC development team</a>
                 (<a href="https://github.com/enwikipedia-acc/waca/issues">Bug reports</a>)
-            </small>
-        </p>
-        <p class="col-md-6 text-right">
-            <small>
+            </p>
+            <ul>
+                <li><a href="{$baseUrl}/internal.php/privacy">Privacy Statement</a></li>
```

Comment:
So what you're saying is that it works just fine in my dev environment and will work fine in prod, but will break horribly on accounts-dev.w.o/rc/?

Reply:
Well, it "works", but perhaps not quite as intended because the configured baseUrl won't be respected; it's an absolute path, but it will just be appended to whatever host the browser sees. For example, while I'm not sure if the dev instances still work this way, this won't actually work at all if ACC is hosted from a subdirectory.

Reply:
For example, if an ACC instance is hosted at `https://accounts-dev.wmflabs.org/fastlizard4/`, clicking the link should direct you to `https://accounts-dev.wmflabs.org/fastlizard4/internal.php/privacy`, but it would instead direct you to `https://accounts-dev.wmflabs.org/internal.php/privacy` as currently implemented.

Reply:
Yep, that's how the dev instances work - from a subdir. Prod and my local dev instance are at the domain root though, which is probably why I didn't spot this.

Reply:
Oh, I just realized that I basically said exactly what you said. Sorry, I'm a bit tired and have goldfish brain. 😅